src/promptflow/promptflow/_sdk/_configuration.py

```diff
@@ -64,6 +75,7 @@ def get_instance(cls):
 
     def set_config(self, key, value):
         """Store config in file to avoid concurrent write."""
+        self._validate(key, value)
```

Comment:
add validate in init if overrides exists

Reply:
good catch, added.